lib/state/ui/reducer.ts

```diff
@@ -11,6 +11,19 @@ const filteredNotes: A.Reducer<T.NoteEntity[]> = (
   action
 ) => ('FILTER_NOTES' === action.type ? action.notes : state);
 
+const listTitle: A.Reducer<string> = (state = 'All Notes', action) => {
```

Comment:
We can ignore it, but I used `T.TranslatableString` to start keeping track of things we'll need to translate later. Might be handy to keep around

Reply:
Done